src/System.Private.CoreLib/src/System/Threading/TimerQueue.Unix.cs

```diff
@@ -111,12 +105,6 @@ private void TimerThread()
             }
         }
 
-        private static int TickCount
-        {
-            get
-            {
-                return Environment.TickCount;
-            }
-        }
+        private static int TickCount => Environment.TickCount;
```

Comment:
Nit: Just call `Environment.TickCount` directly in the two places? This looks like a left over from times when Environment did not live in CoreLib.

Reply:
no, the property is also called from shared part